libs/ui/src/hooks/use_usb_drive.ts

```diff
@@ -55,11 +55,11 @@ export function useUsbDrive({ logger }: UsbDriveProps): UsbDrive {
         await logger.log(LogEventId.UsbDriveEjected, currentUser, {
           disposition: 'failure',
           message: 'USB Drive failed when attempting to eject.',
-          error: error.message,
+          error: (error as Error).message,
```

Comment:
why a diff pattern here?

Reply:
Had done this everywhere, then changed. I'll fix.